### DISCUSSION TITLE
Is it possible to use both React and Vue in the site with Vitesse?

### DISCUSSION THREAD

**Discussion by 684efs3** [antfu/vitesse#269] [Q&A]
Is it possible to use both React and Vue in the site with Vite? I have some components that are pre-built from an external library in React. My site is in Vue, and I would like to pull these React components in. Is this possible in Vite?

I saw that (Astro)[https://astro.build/], which uses Vite can do this, and so was wondering how to do this in a fresh Vitesse install. I tried install both React and Vue and got a black page.

Thanks a lot!

**Comment by thuongtruong1009** [comment]
Hi @684efs3 !
- You can set up React with Vite when you init your app.
![image](https://user-images.githubusercontent.com/71834167/149921367-f17539b5-a55b-4cd3-80be-082e00a497fb.png)
- See more information at [here](https://www.section.io/engineering-education/creating-a-react-app-using-vite/)
- Hope it helps you!
- If can, you can mark it as an answer for another guy to know about this.
- Thank you!
